### PR TITLE
Fixed an issue where running **validate**, **update-release-notes** o…

### PR DESCRIPTION
**Pull request by bakatzir**
…utside of content repo crushed without a meaningful error message

## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/28369

## Screenshots
Screen Shot 2020-09-22 at 9 14 22

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -836,10 +839,14 @@ def update_pack_releasenotes(**kwargs):
...
-    validate_manager.setup_git_params()
-    modified, added, old, changed_meta_files, _packs = validate_manager.get_modified_and_added_files('...',
-                                                                                                     'origin/master')
+    try:
+        validate_manager = ValidateManager(skip_pack_rn_validation=True)
+        validate_manager.setup_git_params()
+        modified, added, old, changed_meta_files, _packs = validate_manager.get_modified_and_added_files(
+            '...', 'origin/master')
+    except (git.InvalidGitRepositoryError, git.NoSuchPathError, FileNotFoundError):
+        print_error("You are running demisto-sdk validate not in the content repository.")
```
Comment (roysagi) [inline demisto_sdk/__main__.py:848]:
```suggestion
        print_error("You are not running `demisto-sdk update-release-notes` command in the content repository.")
```

File: demisto_sdk/__main__.py [hunk 2]
```diff
@@ -285,23 +286,25 @@ def validate(config, **kwargs):
...
+                                        print_ignored_errors=kwargs['print_ignored_errors'],
+                                        is_external_repo=is_external_repo,
+                                        print_ignored_files=kwargs['print_ignored_files'],
+                                        no_docker_checks=kwargs['no_docker_checks'],
+                                        silence_init_prints=kwargs['silence_init_prints'],
+                                        skip_dependencies=kwargs['skip_pack_dependencies'],
+                                        id_set_path=kwargs.get('id_set_path'))
+            return validator.run_validation()
+        except (git.InvalidGitRepositoryError, git.NoSuchPathError, FileNotFoundError):
+            print_error("You are running demisto-sdk validate not in the content repository.")
```
Comment (roysagi) [inline demisto_sdk/__main__.py:307]:
```suggestion
            print_error("You are running not `demisto-sdk validate` in the content repository.")
```

maybe add some guidance as to how to solve this?

Comment (bakatzir) [reply demisto_sdk/__main__.py:307]: ok

File: demisto_sdk/__main__.py [hunk 3]
```diff
@@ -285,23 +286,26 @@ def validate(config, **kwargs):
...
+                                        is_external_repo=is_external_repo,
+                                        print_ignored_files=kwargs['print_ignored_files'],
+                                        no_docker_checks=kwargs['no_docker_checks'],
+                                        silence_init_prints=kwargs['silence_init_prints'],
+                                        skip_dependencies=kwargs['skip_pack_dependencies'],
+                                        id_set_path=kwargs.get('id_set_path'))
+            return validator.run_validation()
+        except (git.InvalidGitRepositoryError, git.NoSuchPathError, FileNotFoundError):
+            print_error("You are not running `demisto-sdk validate` command in the content repository.\n"
+                        "Please run `cd content` from your terminal and run the command again")
```
Comment (yuvalbenshalom) [inline demisto_sdk/__main__.py:308]:
```suggestion
                        "Please run the command from content directory")
```

running `cd content` will not get you to the content dir in most cases.

File: demisto_sdk/__main__.py [hunk 4]
```diff
@@ -285,23 +286,26 @@ def validate(config, **kwargs):
...
+                                        print_ignored_errors=kwargs['print_ignored_errors'],
+                                        is_external_repo=is_external_repo,
+                                        print_ignored_files=kwargs['print_ignored_files'],
+                                        no_docker_checks=kwargs['no_docker_checks'],
+                                        silence_init_prints=kwargs['silence_init_prints'],
+                                        skip_dependencies=kwargs['skip_pack_dependencies'],
+                                        id_set_path=kwargs.get('id_set_path'))
+            return validator.run_validation()
+        except (git.InvalidGitRepositoryError, git.NoSuchPathError, FileNotFoundError):
+            print_error("You are not running `demisto-sdk validate` command in the content repository.\n"
```
Comment (yuvalbenshalom) [inline demisto_sdk/__main__.py:307]:
```suggestion
            print_error("You are not running `demisto-sdk validate` command in the content directory.\n"
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
need to change message a bit

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
no way to unit test this? see coverage.

**Comment by bakatzir** [comment]
@yuvalbenshalom drop in coverage is coveralls issue, see in the related files.
re thinking it however, I can test it. adding.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**